src/main/java/seedu/address/logic/commands/mode/SwitchCommand.java

```diff
@@ -12,12 +12,14 @@
 
     public static final String COMMAND_WORD = "switch";
 
-    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Switches mode. Parameters: "
-            + ModeEnum.getModeOptions()
-            + " \n"
-            + "Example: " + COMMAND_WORD + " contact";
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Switches mode.\n"
+            + "Parameters: MODE (must be " + ModeEnum.getModeOptions()
+            + ")\n"
+            + "Example: " + COMMAND_WORD + " " + ModeEnum.DELIVERABLE.getArgument();
+
 
     public static final String MESSAGE_SUCCESS = "Mode switched to: %1$s";
```

Comment:
Does this mean users will get something like `Mode switched to: dv`? Are we able to convert into `Mode switch to: deliverable` (along with the other models) instead? 😨

Reply:
Yes, its currently Mode switched to: Deliverable